full-stack-tests/backend/src/perftest/SchemaLoader.test.ts

```diff
@@ -0,0 +1,138 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import * as path from "path";
+import { Reporter } from "@itwin/perf-tools";
+import { IModelJsFs, KnownLocations, StandaloneDb } from "@itwin/core-backend";
+import { IModelTestUtils, KnownTestLocations } from "@itwin/core-backend/lib/cjs/test";
+
+describe("SchemaLoaderPerformance", () => {
+  const outDir: string = path.join(KnownTestLocations.outputDir, "SchemaLoaderPerformance");
+  const reporter = new Reporter();
+
+  before(() => {
+    if (!IModelJsFs.existsSync(KnownTestLocations.outputDir))
+      IModelJsFs.mkdirSync(KnownTestLocations.outputDir);
+    if (!IModelJsFs.existsSync(outDir))
+      IModelJsFs.mkdirSync(outDir);
+  });
+
+  after(() => {
+    const csvPath = path.join(outDir, "SchemaLoaderPerfResults.csv");
+    reporter.exportCSV(csvPath);
+  });
+
+  function getSchemaPath(domain: string, schemaFileName: string): string {
+    return path.join(KnownLocations.nativeAssetsDir, "ECSchemas", domain, schemaFileName);
+  }
+
+  function getSchemaPathFromPackage(packageName: string, schemaFileName: string): string {
+    const schemaFile = path.join(__dirname, "..", "..", "..", "node_modules", "@bentley", packageName, schemaFileName);
+    return schemaFile;
+  }
+
+  async function timeBisSchemasLoading(schemaName: string, schemaPath: string): Promise<void> {
+    const rootSubject = { name: "Performance tests", description: "Performance tests" };
+    const snapshotFile: string = IModelTestUtils.prepareOutputFile("Performance", "Performance.bim");
+    const imodel: StandaloneDb = StandaloneDb.createEmpty(snapshotFile, { rootSubject });
+
+    try {
+      await imodel.importSchemas([schemaPath]);   // will throw an exception if import fails
+    } catch (error) {
+      throw new Error(`Failed to import schema ${schemaPath}`);
+    }
+
+    imodel.saveChanges();
+
+    const startTime: number = new Date().getTime();
```

Comment:
you need to close and reopen the connection after importing the schema to clear the cache.